.gitlab-ci.yml

```diff
@@ -0,0 +1,65 @@
+include:
+  - local: '.gitlab-ci-check-commits.yml'
+  - local: '.gitlab-ci-check-license.yml'
```

Comment:
Sorry I didn't explain clear enough. It is only `mendertesting` which has all these extra pipelines which are used from non-golang repos.

This PR should only add one pipeline (this file, `.gitlab-ci.yml`).

Correct the commit message accordingly.

Reply:
Done